api/src/certification/course/application/certification-course-route.js

```diff
@@ -57,6 +57,21 @@ const register = async function (server) {
       method: 'POST',
       path: '/api/admin/certification-courses/{id}/assessment-results',
       config: {
+        validate: {
+          params: Joi.object({
+            id: identifiersType.certificationCourseId,
+          }),
+          payload: Joi.object({
+            data: {
+              attributes: {
+                'comment-for-organization': Joi.string().allow(null, '').required(),
+                'comment-for-candidate': Joi.string().allow(null, '').required(),
+                'comment-by-jury': Joi.string().allow(null, '').required(),
+              },
+              type: Joi.string(),
```

Comment:
```suggestion
              type: Joi.string().valid('results'),
```
Je ne suis pas certain du type, à tester

Reply:
Je supprime le type, il n'apporte rien

Reply:
Done